src/main/java/io/cdap/plugin/salesforce/parser/SalesforceQueryVisitor.java

```diff
@@ -148,7 +148,7 @@ public SObjectDescriptor.FieldDescriptor visitFieldName(SOQLParser.FieldNameCont
         // select Opportunity.Name from Opportunity -> Name
         // select o.Name from Opportunity o -> Name
         String topParent = nameParts.get(0);
-        if (topParent.equalsIgnoreCase(objectName) || topParent.equals(objectAlias)) {
+        if (topParent.equals(objectName) || topParent.equals(objectAlias)) {
```

Comment:
why do we need this change to it case sensitive?

Reply:
There was a bug in the Salesforce connector:  [PLUGIN-1184](https://cdap.atlassian.net/browse/PLUGIN-1184) in April this year where we added support for the case in-sensitivity in the SOQL query.

Now, it would work both for the lower case as well as upper case characters.